lib/router/unrecognized-url-error.js

```diff
@@ -5,9 +5,25 @@ import { oCreate } from './utils';
   handlers for failed transitions.
  */
```

Comment:
Hehe, I agree it seems pretty whacky.  Mind either removing it completely or adding something that describes reality?